### PR TITLE
Get random word

### PR DESCRIPTION
**Pull request by RyanSelesnik**
**Acceptance tests**

The random word returned each day should be a valid word in the dictionary

### CODE REVIEW

**Comment by RyanSelesnik** [comment]
closes #59

**Comment by GiaCroock** [comment]
Besides for the additional test that needs to be added, your code is clear, easy to understand, conforms to StandardJS guidelines. For a future improvement the new word of the day should only be done through a post request. however, for the purpose of your user story this function meets the requirements.

File: src/__tests__/words.spec.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const dictionary = require("../models/dictionary");
+const validWords = dictionary.getDictionary();
+
+describe("When making a request to /word", () => {
+  it("should return a valid word", async () => {
+    const response = await request(app).get("/");
+    word = response.body;
+    expect(response.statusCode).toBe(200);
+    expect(validWords.includes(word)).toBe(true);
+  });
```
[the remark this answers is not in the document]
Comment (GiaCroock) [reply src/__tests__/words.spec.js:21]: Specifically that a new word should be added per day

**Review by GiaCroock — CHANGES_REQUESTED** [review]
Add additional test to check new word generated each day

**Comment by GiaCroock** [comment]
The new test has been implemented and works. Everything else looks good.

**Pull request merged**